apps/application-system/api/src/app/modules/payment/utils/findItemType.ts

```diff
@@ -0,0 +1,74 @@
+//import { DrivingLicenseType } from '../types/schema'
+
+export const findItemType = (itemCode: string) => {
+  let typeId = itemCode
+  // AYXXX PayableDummyTemplate should be deleted when Dummy testing has finished.
```

Comment:
it's good practice to prefix such comments with `// TODO: ` - a lot of editors can present those in its own panel etc and there's some additional tooling around such things